api/core/index.qmd

```diff
@@ -1,3 +1,7 @@
+---
+repo-actions: false
```

Comment:
Thanks for trying this! This will work for this page, but not for any others under `api/`. I think the solution will be to place this setting in `api/_metadata.yml` file so that it applies to the whole subtree.

The likely challenge with that approach is that we programmatically build the files under `api/` via quartodoc and there's a good chance that we clean out `api/` entirely with each build. So I'm not sure where that will leave us.

Reply:
looks like putting it in a meta file should do it:
<img width="1315" alt="image" src="https://github.com/user-attachments/assets/50f2f1c6-24ec-481a-becc-aa11348e9537">